src/components/app/TabBar.css

```diff
@@ -17,20 +17,23 @@
 
 .tabBarTabWrapper {
   display: flex;
+  overflow: auto hidden;
   min-width: 0; /* This makes the tab container actually shrinkable below min-content */
+  flex: 1;
   flex-flow: row nowrap;
   padding: 0;
-  margin: 0 -0.5px; /* This combines with the -0.5px horizontal margin of .tabBarTab to clip off the first tab's 1px starting border */
+  margin: 0 -0.5px -1px; /* This combines with the -0.5px horizontal margin of .tabBarTab to clip off the first tab's 1px starting border */
```

Comment:
New comment:

`The -0.5px horizontal margin combines with the -0.5px horizontal margin of .tabBarTab to clip off the first tab's 1px starting border. The -1px bottom margin makes our contents cover the .Details-top-bar bottom border.`